test/py/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ function(add_py_test NAME SCRIPT FIXTURE_NAME VENV_DIR)
             "PYTHONMALLOC=debug"
             "MALLOC_CHECK_=3"
         )
-        if(ENABLE_VIRTUAL_ENV)
+        if(NOT MIGRAPHX_DISABLE_VIRTUAL_ENV)
```

Comment:
Suggestion : Better to flip if and else blocks

Reply:
Thanks for the suggestion. I have made the change.